components/pingdom/sources/new-check/test-event.mjs

```diff
@@ -0,0 +1,12 @@
+export default {
+	"checkid": 123123,
+	"time": 123123123123,
+	"via": "email",
+	"status": "sent",
+	"messageshort": "down",
+	"messagefull": "Message alert",
+	"sentto": "John@doe.com",
+	"charged": false,
+	"username": "John Doe",
+	"userid": 123123
+}
```

Comment:
Are you sure this is the correct test event for `checks`? The documentation looks a little different.

      "type": "http",
      "id": 100,
      "name": "mycheck",
      "lasterrortime": 1297446423,
      "lasttesttime": 1300977363,
      "lastresponsetime": 355,
      "lastdownstart": 1300977363,
      "lastdownend": 1300977463,
      "status": "up",
      "resolution": 0,
      "hostname": "string",
      "created": 0,
      "tags": [
        {
          "name": "apache",
          "type": "a",
          "count": 2
        }
      ],
      "ipv6": true
    }